test/TPP/linalg-to-tpp.mlir

```diff
@@ -51,29 +51,3 @@ func.func @matmul_lowering(%arg0: memref<8x9xf32>,
                 outs(%arg2: memref<8x8xf32>)
   return
 }
-
-// -----
-
-#map5 = affine_map<(d0, d1, d2) -> (d0, d1, d2)>
```

Comment:
Why was this test dropped? Do we not recognise this any more as `tpp.relu`?

Reply:
Actually, this looks like a repeat of the test above, is that the reason?

Reply:
No simply because we don't run the detection in the conversion pass anymore. We revert to original behavior where we first need to run `-map-linalg-to-tpp` and then `convert-linalg-to-tpp`. It is better to keep these as separate passes.

Reply:
Is there a similar test elsewhere, then?

Reply:
Yes, the one above. Was the same only without the `tpp.relu` annotation.

Reply:
I see. Do we have tests that check that the mapping happens for all cases?

Reply:
with all cases you mean `maxf(0, x)` and `maxf(0, x)`? If so, no. And the mapping is quite not right at the moment as we only check to have a `maxf`.

Reply:
Sorry, no. `(0, x)` can be normalised to `(x, 0)` and we're good.

This test seems to look at all "recogniseable" linalg ops that have the TPP shape (or annotation). IIUC, that's done by the mapping pass.

Do we have tests for the mapping pass, to make sure that test that you removed will continue to be recognised in the future? 

Here, we only have hand-coded IR that already has the annotation, I'm wondering if we have tests that make sure we keep adding annotation to the right constructs.

Reply:
>  And the mapping is quite not right at the moment as we only check to have a maxf

So, if we have a `maxf(x, 2)` it will map to RELU?

Reply:
We don't have much: https://github.com/plaidml/tpp-sandbox/blob/main/test/TPP/simple-match.mlir and yes at the moment `maxf(x, 2)` will get annotated with a tpp.relu

Reply:
We also need to make the relu single operand see: https://github.com/plaidml/tpp-sandbox/issues/129

Reply:
> We don't have much: https://github.com/plaidml/tpp-sandbox/blob/main/test/TPP/simple-match.mlir

That should be fine for the time being.

> and yes at the moment `maxf(x, 2)` will get annotated with a tpp.relu

#130